src/libraries/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.DivRem.cs

```diff
@@ -138,55 +124,48 @@ public static unsafe uint[] Remainder(uint[] left, uint[] right)
 
             // NOTE: left will get overwritten, we need a local copy
 
-            uint[] localLeft = CreateCopy(left);
+            uint[] localLeft = left.ToArray();
```

Comment:
This allocates. The local copy of left, that is used in `Divide` can be stack-alloced / ArrayPooled.

Reply:
`localLeft` escapes the method. Not possible to use stack alloc or pooling.

Reply:
IIRC (I'm on mobile now, so hard to check) this calls `Divide` which takes span-arguments, so why you can't pass a span around? Similar to https://github.com/dotnet/runtime/pull/35565#discussion_r417630924